### ISSUE TITLE
Все отзывы

### ISSUE THREAD

**Issue by iliakan** [iliakan/javascript-nodejs#903]
Обнаружил, что когда заходишь на главную страницу курса и листаешь слайдер отзывов (ты недавно апдейтил дизайн), то, когда упираешься в конец (их всего 5 штук), то не можешь найти как посмотреть другие отзывы.

Ссылка на "все отзывы" есть в дизайне, но глаз её не видит, т.к. там много элементов.

Может, что-то вместо правой стрелки сделать, или как-то ещё решить?

**Comment by Bezart** [comment]
Что думаешь о таком варианте http://take.ms/4KKJ7 ? У нас уже есть хороший суммарный модуль, можно его использовать как дополнительный 6й слайд для того чтобы заинтересовать пользователя

**Comment by iliakan** [comment]
Классное решение.

**Comment by iliakan** [comment]
@Bezart имеет в виду http://learn.javascript.ru/courses/js/feedback насчет суммарного модуля.  Диаграмма там рисуется через JS так:
```

    +b.pie
      +e('svg').image(height="180" width="180")
        circle(cx=90 cy=90 r=90 fill="#469269")
        path(fill="#010A05" id="statsPercentagePath" fill-opacity="0.22")

      script
        :uglify
          !function() {
            function drawPercentage(percentage) {
              var r = 90;
              var cx = 90;
              var cy = 90;

              // normal formula is:
              // cx + r * Math.cos(2PI*percentage)
              // the angle is calculated clockwise & starting from horizontal X-axis
              // so we substract PI/2 to make it start 1/4-circle earlier
              var x = cx + r * Math.cos(2*Math.PI*percentage - Math.PI/2);
              var y = cy + r * Math.sin(2*Math.PI*percentage - Math.PI/2);

              var d = "M" + x + "," + y + " A90,90 0 " + (percentage >= 0.5 ? 0 : 1) + ",1 90,0 L90,90 z";
              document.getElementById('statsPercentagePath').setAttribute('d', d);
            }

            drawPercentage(feedbackStats.recommendFraction);
          }();
```

То есть, мини-svg прям в HTML и через JS в `path` ставится `d`, можно полный мини-svg увидеть на странице выше инспектом.

В вёрстке JS не нужен, просто статично написать расчёт такой же, будут сложности - пиши, мне, наверно, проще код для круга адаптировать, т.к. его я делал.

**Comment by iliakan** [comment]
@bezart, а на основной курсов такого нет, да? Там, где с каждого курса по 1 отзыву. Просто конец? (у нас нет общей страницы по всем курсам, на каждый свои отзывы).

**Comment by Bezart** [comment]
Да, для общего блока это не сработает. Можно в принципе показывать список ссылок на все отзывы для всех доступных курсов.

Курс 1 – все отзывы (12)
Курс 2 – все отзывы (35)
....

Как тебе такое решение?

**Comment by iliakan** [comment]
Можно.. Сейчас 5 курсов. Потом как больше будет - переделаем?

**Comment by dzhiriki** [comment]
@Bezart @iliakan 
А показывать ссылки где? В смысле, будет такой же 6 слайд, но вместо одной ссылки «все отзывы о курсе», ссылок будет несколько?
Или я не так понял?

**Comment by iliakan** [comment]
@dzhiriki да, совершенно правильно, дополнительный отдельный слайд с этой информацией после отзывов.

**Comment by Bezart** [comment]
@iliakan может со средними рейтингами http://take.ms/4pa1w ?

**Comment by iliakan** [comment]
Рейтинги подразумевают сравнение курсов, оно здесь не уместно, курсы разные.

Числа лучше опубликовать, когда их будет больше, по react там вообще без отзывов курсы велись, получится неправильно, как будто групп не было.

**Comment by dzhiriki** [comment]
Если оставлять такой же блок, как и 6 слайд на странице курса, то непонятно что должно быть в заголовке (на странице курса там название курса).

Ну и в целом, по мне так, общая статистика всех отзывов имеет достаточно мало смысла — рейтинги по разным курсам могут быть очень разные. А тут этакое среднее по больнице.

**Comment by iliakan** [comment]
@dzhiriki этот твой вопрос - он по странице где все курсы, да?

Может, пока на странице одного курса такой блок сделаем?

**Comment by iliakan** [comment]
А вот это "Курс по TypeScript" можно сделать ссылкой на курс? Не меняя дизайна.

**Comment by iliakan** [comment]
(т.е. при наведении выделять только)

**Comment by dzhiriki** [comment]
Которое на слайде с суммарной инфой?
Этот блок же и так отображается на странице курса, зачем ему быть ссылкой?

**Comment by iliakan** [comment]
На странице где все курсы, когда смотришь отзывы, то если заинтересовал курс, хочется сразу к нему перейти.
Навожу на заголовок... а вот :(

Это даже больше вопрос к @Bezart

**Comment by iliakan** [comment]
@bezart ещё вопрос: цвет посещённой ссылки некрасиво смотрится, вот:
http://jmp.sh/xQ1tDUI

**Comment by iliakan** [comment]
@bezart:
1) цвет посещённой ссылки
2) название курса как ссылка при перелистывании отзывов на странице всех курсов
3) на странице всех курсов в итоге без summary-слайда?

**Comment by Bezart** [comment]
@iliakan 

1) не думаю, что нужно менять цвет посещенной ссылки для этого случая. По-моему смотрится не хуже чем в других местах. Большинство пользователей будут видеть ссылку не посещенной

2) Слишком много ссылок как для одного модуля. Найти ссылку на сам курс можно и со страницы отзыва, а вот куда ведет ссылка в заголовке не очевидно, так как контекст здесь про отзывы. По умолчанию, я бы подумал, что ссылка будет на все отзывы по курсу. Хотелось бы сфокусироваться на основной цели этого блока — предоставить реальные доказательство качества курсов.

3) Можно и без самари
